VS_Meadow_Extension/VS_Meadow_Extension.Shared/MeadowPackage.cs

```diff
@@ -67,13 +67,16 @@ protected override async Task InitializeAsync(CancellationToken cancellationToke
         {
             await base.InitializeAsync(cancellationToken, progress);
 
+            // Make settings loading lazy
+            meadowSettingsLazy = new Lazy<MeadowSettings>(() => new MeadowSettings(Globals.SettingsFilePath));
+
+            // Ensure Install dependencies is off loaded to a background thread
+            await Task.Run(() => InstallDependencies(cancellationToken), cancellationToken);

```

Comment:
remove Task.Run and replace with _ = InstallDep....

Reply:
Done.